requirements.txt

```diff
@@ -21,6 +21,7 @@ raven[flask]==5.8.1
 ujson==1.33
 pyelasticsearch==1.4
 requests==2.10.0
+git+https://github.com/timClicks/slate.git
```

Comment:
the pypi version doesn't work :( Known issue that has been unresolved for some time.